docs/src/rust/user-guide/expressions/structs.rs

```diff
@@ -95,5 +95,54 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     println!("{}", &out);
     // --8<-- [end:struct_ranking]
 
+    // --8<-- [start:multi_column_apply]
+    let df = df!(
+        "keys" => &["a", "a", "b"],
+        "values" => &[10, 7, 1],
+    )?;
```

Comment:
why not use the same values as in the Python example (a, a, b, b; 10, 7, 1, 23)?